support-owasp/src/main/resources/dependency-check-maven-config.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<suppressions xmlns="https://www.owasp.org/index.php/OWASP_Dependency_Check_Suppression">
```

Comment:
❓ Should we move these suppression to DDF?  Can we guarantee that all of these libraries of these versions will not be used in a safe way for all Codice projects?

Reply:
Probably should be as close as possible to the areas they are suppressed in to ensure we aren't blanket covering future usages.